README.md

```diff
@@ -23,7 +23,7 @@ def application do
 end
 
 def deps do
-  [{:slack, "~> 0.9.3"}]
+  [{:slack, "~> 0.9.4"}]
```

Comment:
Can you leave the version bump out? I'll handle this in a separate commit so I can do tagging and etc.